Makefile

```diff
@@ -115,11 +118,12 @@ start: install
 	$(INSTALL_DIR)/bin/python manage.py runserver $(PORT) --settings=tests.settings
 
 # Lint the project
-lint: clean_working_directory
+lint: clean_working_directory $(INSTALL_DIR)/bin/flake8
 	$(call header,"Linting code")
-	@find . -type f -name '*.py' -not -path '$(INSTALL_DIR)/*' -not -path './docs/*' -not -path './build/*' | xargs flake8
+	@find . -type f -name '*.py' -not -path '$(INSTALL_DIR)/*' -not -path './docs/*' -not -path './build/*' | xargs $(INSTALL_DIR)/bin/flake8
+	@. $(INSTALL_DIR)/bin/activate; isort --check-only --recursive $(APP_NAME) tests
```

Comment:
The confusion comes from line 79, where you do `lint` before `install` so that `flake8` is not available yet. Thus I added manually. Will fix it then and remove manual flake8 installation.